ansible_runner/streaming.py

```diff
@@ -87,8 +87,12 @@ def update_paths(self, kwargs):
 
     def run(self):
         while True:
-            line = self._input.readline()
-            data = json.loads(line)
+            try:
+                line = self._input.readline()
```

Comment:
Wasn't the problem seen that `line` is empty string? If that _should_ be an error case, I'd like to have a special error message just for that case.

Reply:
I think so.  This was coming about if I remember correctly because the sender was crashing, so we were receiving less data than we were expecting.